pkg/controller/controller.go

```diff
@@ -70,10 +69,8 @@ type Controller struct {
 }
 
 type Config struct {
-	Namespace     string
-	MasterHost    string
-	KubeCli       *unversioned.Client
-	PVProvisioner string
+	MasterHost string
+	cluster.Config
```

Comment:
this seems to be a wrong abstraction. Controller config should not include a cluster specific config.